content/applications/hr/fleet/accidents.rst

```diff
@@ -0,0 +1,257 @@
+the safest drivers.
+
+Due to the customizable nature of Odoo's *Fleet* app, there are several different ways to track
+accidents. The instructions below provide step by step instructions for only **one** method to keep
+track of accidents and their subsequent repair costs.
+
+Structure
+=========
+
+For this example, to track accidents, two :ref:`service types <fleet/new-type>` are created:
+`Accident - Driver's Fault` and `Accident - No Fault`.
+
+This will make it possible to track the various repairs associated with accidents, organized by the
+type of accident (who was at fault).
+
+When an accident occurs, a service record is created. The specific repairs needed for the accident
+are logged in the *description* of the service record, and the details about the accident (what
+happened) are logged in the *notes* section.
+
+With this organizational structure, it is possible to view all accidents organized by fault, car,
+driver, or cost.
+
+.. seealso::
+   To manage accidents, the creation of service records is **required**.
+
+   Refer to the :doc:`service` documentation for detailed instructions on creating service records
+   in Odoo's *Fleet* app.
+
+Log accidents and repairs
+=========================
+
+After an accident occurs, repairs are usually needed. To log an accident, the first step is to
+:ref:`create a service record <fleet/service-form>` detailing the specific repairs needed.
+
+.. note::
+   Some accidents require multiple repairs with several different vendors. For these scenarios, a
+   separate service record is needed for each vendor performing repairs. To keep records organized,
+   it is recommended to keep the :guilabel:`NOTES` field identical, as well as attaching the same
+   important documentation, such as a police report.
+
+Navigate to :menuselection:`Fleet app --> Fleet --> Services` to view the main :guilabel:`Services`
+dashboard. Click :guilabel:`New` in the top-left corner and a blank service form loads.
+
+Enter the following information on the form:
+
+- :guilabel:`Description`: enter the description of repairs needed to fully repair the vehicle, such
+  as `Bodywork`, `Windshield Replacement`, or `Replacement Bumper, Tires, and Windows`.
+- :guilabel:`Service Type`: select either `Accident - Driver's Fault` or `Accident - No Fault`,
+  depending on the situation.
+
+  When entering either of these two :guilabel:`Service Types` for the first time, enter the new
+  service type, then click :guilabel:`Create (new service type)`.
+
+  Once an accident service type has been added to the database, it is available to select from the
+  drop-down menu in the :guilabel:`Service Type` field.
+- :guilabel:`Date`: using the calendar popover window, select the date the accident occurred.
+  Navigate to the desired month using the :icon:`fa-chevron-left` :icon:`fa-chevron-right`
+  :guilabel:`(chevron)` icons, then click on the date to select it.
```

Comment:
This is my personal opinion, so feel free to ignore this comment, but even though "chevron" is the technical term for the icon, I think referring to it as an arrow icon would ensure the icon is most widely understood.
```suggestion
  :guilabel:`(arrow)` icons, then click the date to select it.
```